kura/org.eclipse.kura.asset.provider/src/main/java/org/eclipse/kura/asset/provider/BaseAsset.java

```diff
@@ -95,8 +87,8 @@
     /** The provided asset configuration wrapper instance. */
     private AssetConfiguration assetConfiguration;
 
-    /** Container of mapped asset listeners and drivers listener. */
-    private final Map<AssetListener, DriverListener> assetListeners;
+    /** Container of channel listeners registered by this Asset. */
```

Comment:
Registered by this asset to the attached driver.
The channel listeners now are registered directly forwarded to the driver, the only reason for having this set is to remove all channel listeners registered by this asses at the deactivate step.